testsuite/features/init_clients/buildhost_bootstrap.feature

```diff
@@ -60,11 +59,3 @@ Feature: Bootstrap a build host via the GUI
   Scenario: Check events history for failures on SLES build host
     Given I am on the Systems overview page of this "build_host"
     Then I check for failed events on history event page
-
```

Comment:
We had to do it before this PR otherwise all onboarded minions would have the build host and OS image build host attribute, since we only use one key for onboarding.